webpack/webpack.common.js

```diff
@@ -24,7 +24,7 @@ module.exports = {
     },
     context: path.resolve(__dirname, "../src"),
     resolve: {
-        extensions: [".js", ".jsx"],
+        extensions: [".js", ".jsx", ".md", ".mdx"],
```

Comment:
Just double checking and I don't know the answer here... is it necessary to include the md and mdx extensions here?

Reply:
I made the change here rather than trying to change line 33 in AboutTheDataDrilldown to preload both .md and .mdx